website/docs/r/azure_metrics_source.html.markdown

```diff
@@ -70,6 +72,8 @@ resource "sumologic_collector" "collector" {
 ## Argument reference
 In addition to the [Common Source Properties](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs#common-source-properties), the following arguments are supported:
 - `content_type` - (Required) Must be `AzureMetrics`.
+- `scan_interval` - (Optional) Time interval in milliseconds of scans for new data. The default is 300000 and the minimum value is 1000 milliseconds.
+- `paused` - (Optional) When set to true, the scanner is paused. To disable, set to false. The default is false.
```

Comment:
we don't need to add `paused` field, the pausing capability is currently not available for the azure metrics source type